exercises/slot_machine.livemd

```diff
@@ -11,10 +11,13 @@ Create a `SlotMachine` [GenServer](https://hexdocs.pm/elixir/GenServer.html) tha
 
 When the `SlotMachine.roll/1` function is called the user should have a one in three chance of getting a reward.
 
-The should be between `1` and the remaining balance of the machine.
+The reward should be between `1` and the remaining balance of the machine.
 
 * use `:sys.get_state/1` to make sure that the `SlotMachine` is handling its state properly.
```

Comment:
Capitalization, also is this a command for the student, or a suggestion?

It feels like it should be a suggestion.

"You can use `:sys.get_state/1` to debug the `SlotMachine`'s state."